vm/src/tests/cairo_pie_test.rs

```diff
@@ -269,7 +266,11 @@ fn serialize_and_deserialize_cairo_pie() {
     let result = runner.get_cairo_pie(&vm);
     assert!(result.is_ok());
     let cairo_pie = result.unwrap();
-    let cairo_pie_serialized = serde_json::to_string(&cairo_pie).unwrap();
-    let cairo_pie_deserialized: CairoPie = serde_json::from_str(&cairo_pie_serialized).unwrap();
-    assert_eq!(cairo_pie_deserialized, cairo_pie);
+
+    // TODO: Find a way to test the entire object, and not just `metadata`.
```

Comment:
Aren´t we checking the other fields, like `memory`, `additional_data`. etc

Reply:
Yes, I forgot to remove it. Fixed, thanks.